code/controllers/configuration/sections/movement_configuration.dm

```diff
@@ -9,7 +9,7 @@
 	/// Move delay for humanoids
 	var/human_delay = 1.5
 	/// Move delay for cyborgs
-	var/robot_delay = 2.5
+	var/robot_delay = 1.5
```

Comment:
to update?

Reply:
This is why there's the extra ", = . - 1" later, since this is the config change that isn't getting properly added to the TM.